diff -urN libgda-1.9.103.org/libgda/Makefile.am libgda-1.9.103/libgda/Makefile.am
--- libgda-1.9.103.org/libgda/Makefile.am	2006-08-29 15:23:25.000000000 +0200
+++ libgda-1.9.103/libgda/Makefile.am	2006-12-24 16:41:10.073379500 +0100
@@ -208,7 +208,7 @@
 	$(libgda_sources)
 
 libgda_3_0_la_LDFLAGS = -version-info $(GDA_CURRENT):$(GDA_REVISION):$(GDA_AGE) -no-undefined
-libgda_3_0_la_LIBADD = $(LIBGDA_LIBS) $(FAM_LIBS) $(top_builddir)/libsql/libgdasql.la sql-transaction/libsqltransaction.la
+libgda_3_0_la_LIBADD = $(LIBGDA_LIBS) $(FAM_LIBS) $(top_builddir)/libsql/libgdasql.la sql-transaction/libsqltransaction.la $(top_builddir)/libgda/sql-delimiter/libgda_sql_delimiter-3.0.la $(top_builddir)/libgda/handlers/libgda_handlers-3.0.la
 
 gdaincludedir=$(includedir)/libgda-$(GDA_MAJOR_VERSION).$(GDA_MINOR_VERSION)/libgda
 gdainclude_HEADERS=$(libgda_built_headers) $(gda_headers)
