diff -Nru libgda-1.1.2.orig/configure.in libgda-1.1.2/configure.in
--- libgda-1.1.2.orig/configure.in	2004-04-18 13:20:49.000000000 +0200
+++ libgda-1.1.2/configure.in	2004-04-27 09:07:56.527207976 +0200
@@ -70,7 +70,7 @@
 GETTEXT_PACKAGE=libgda-2
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE")
-ALL_LINGUAS="az ca cs da de el en_CA en_GB es fa fi fr ga gl hr hu it ja ko ml ms nl no pl pt pt_BR ru sk sl sr sr@Latn sv tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="az ca cs da de el en_CA en_GB es fa fi fr ga gl hr hu it ja ko ml ms nb nl pl pt pt_BR ru sk sl sr sr@Latn sv tr uk vi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 dnl ******************************
