--- libgda-1.0.4/configure.in.orig	Thu Jul 22 12:09:19 2004
+++ libgda-1.0.4/configure.in	Thu Jul 22 12:13:11 2004
@@ -70,7 +70,7 @@
 GETTEXT_PACKAGE=libgda-2
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE")
-ALL_LINGUAS="az ca cs da de el en_CA en_GB es fa fi fr ga gl hr hu id it ja ko ml ms nl no pl pt pt_BR ru sk sl sr sr@Latn sv tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="az ca cs da de el en_CA en_GB es fa fi fr ga gl hr hu id it ja ko ml ms nl nb pl pt pt_BR ru sk sl sr sr@Latn sv tr uk vi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 dnl ******************************
