diff -urN libgda-0.2.93.org/providers/gda-ldap-server/gda-ldap-connection.c libgda-0.2.93/providers/gda-ldap-server/gda-ldap-connection.c
--- libgda-0.2.93.org/providers/gda-ldap-server/gda-ldap-connection.c	Thu Jan 10 01:59:54 2002
+++ libgda-0.2.93/providers/gda-ldap-server/gda-ldap-connection.c	Thu Jan 10 10:51:11 2002
@@ -284,16 +284,18 @@
 		     GdaServerConnection * cnc, gchar * where)
 {
 	gchar *err_msg;
+	gint err_nr;
 	LDAP_Connection *ld_cnc;
 
 	ld_cnc = (LDAP_Connection *)
 		gda_server_connection_get_user_data (cnc);
 	if (ld_cnc) {
-		err_msg = ldap_err2string (ld_cnc->ldap->ld_errno);
+	    	ldap_get_option(ld_cnc->ldap, LDAP_OPT_ERROR_NUMBER, &err_nr);
+		err_msg = ldap_err2string (err_nr);
 		gda_log_error (_("error '%s' at %s"), err_msg, where);
 
 		gda_error_set_description (error, err_msg);
-		gda_error_set_number (error, ld_cnc->ldap->ld_errno);
+		gda_error_set_number (error, err_nr);
 		gda_error_set_source (error, "[gda-ldap]");
 		gda_error_set_help_url (error, _("Not available"));
 		gda_error_set_help_context (error, _("Not available"));
