diff -aurN libgda-1.3.91.orig/libgda/gda-intl.h libgda-1.3.91/libgda/gda-intl.h
--- libgda-1.3.91.orig/libgda/gda-intl.h	2003-01-16 01:23:28.000000000 +0100
+++ libgda-1.3.91/libgda/gda-intl.h	2005-08-16 13:21:29.000000000 +0200
@@ -26,6 +26,7 @@
 
 #ifdef ENABLE_NLS
 #include<libintl.h>
+#include<locale.h>
 #define _(String) dgettext(GETTEXT_PACKAGE,String)
 #ifdef gettext_noop
 #define N_(String) gettext_noop(String)
