--- libgda-1.9.100/providers/postgres/utils.c.orig	2005-09-22 12:24:01.000000000 +0200
+++ libgda-1.9.100/providers/postgres/utils.c	2006-02-21 20:28:46.267416250 +0100
@@ -289,7 +289,7 @@
 		break;
 	case GDA_VALUE_TYPE_BLOB :
 		blob = gda_postgres_blob_new (cnc);
-		gda_postgres_blob_set_id (GDA_POSTRES_BLOB (blob), atoi (thevalue));
+		gda_postgres_blob_set_id (GDA_POSTGRES_BLOB (blob), atoi (thevalue));
 		gda_value_set_blob (value, blob);
  		break;
 	default :
