diff -urN aaa/libgda-0.91.0/configure.in libgda-0.91.0/configure.in
--- aaa/libgda-0.91.0/configure.in	2003-07-28 23:40:29.000000000 +0200
+++ libgda-0.91.0/configure.in	2003-08-06 16:08:16.000000000 +0200
@@ -31,8 +31,6 @@
 
 AC_PROG_INTLTOOL
 
-GNOME_PLATFORM_GNOME_2(yes, force)
-
 AC_DEFINE_UNQUOTED(BONOBO_EXPLICIT_TRANSLATION_DOMAIN, "libgda-2")
 
 dnl ******************************
@@ -76,7 +74,6 @@
 dnl ******************************
 dnl Checks for libraries
 dnl ******************************
-GNOME_COMPILE_WARNINGS(maximum)
 CFLAGS="$CFLAGS $WARN_CFLAGS"
 CPPFLAGS="$CPPFLAGS $WARN_FLAGS"
 AC_SUBST(CFLAGS)
@@ -354,13 +351,13 @@
 	AC_MSG_CHECKING(for freetds files)
 	for d in $dir /usr/local/freetds /usr /opt/freetds /opt/packages/freetds /opt/freetds-0.50
 	do
-		if test -f "$d/$include/tds.h" -a -e "$d/$lib/libtds.so" -a -f "$d/include/tdsver.h"
+		if test -f "$d/include/tds.h" -a -e "$d/$lib/libtds.so" -a -f "$d/include/tdsver.h"
 		then
 			AC_MSG_RESULT(found freetds in $d)
 			freetdsdir="$d"
-			freetds_incdir="$d/$include"
+			freetds_incdir="$d/include"
 			break
-		elif test -f "$d/$include/freetds/tds.h" -a -e "$d/$lib/libtds.so" -a -f "$d/include/freetds/tdsver.h"
+		elif test -f "$d/include/freetds/tds.h" -a -e "$d/$lib/libtds.so" -a -f "$d/include/freetds/tdsver.h"
 		then
 			AC_MSG_RESULT(found freetds in $d and includes in $d/include/freetds)
 			freetdsdir="$d"
