diff -ur libgda-1.9.103.org/configure.in libgda-1.9.103/configure.in
--- libgda-1.9.103.org/configure.in	2006-08-29 15:23:24.000000000 +0200
+++ libgda-1.9.103/configure.in	2006-12-24 16:23:32.075258750 +0100
@@ -166,7 +166,7 @@
 		else
 			AC_DEFINE(HAVE_BDB, 1, [Have Berkeley DB])
 			BDB_CFLAGS="-I${bdbdir}/include"
-			BDB_LIBS="-L${bdbdir}/lib ${bdblib}"
+			BDB_LIBS="-L${libdir} ${bdblib}"
 		fi
 	fi
 fi
@@ -221,7 +221,7 @@
 	else
 		AC_DEFINE(HAVE_ODBC, 1, [Have ODBC])
 		ODBC_CFLAGS="-I${odbcdir}/include"
-		ODBC_LIBS="-L${odbcdir}/lib ${odbclib}"
+		ODBC_LIBS="-L${libdir} ${odbclib}"
 	fi
 fi
 
@@ -277,7 +277,7 @@
 		else
 			AC_DEFINE(HAVE_MYSQL, 1, [Have MySQL])
 			MYSQL_CFLAGS="-I${mysqldir}/include${mysqldir_suffix}"
-			MYSQL_LIBS="-L${mysqldir}/lib${mysqllibdir_suffix} -lmysqlclient"
+			MYSQL_LIBS="-L${libdir}${mysqllibdir_suffix} -lmysqlclient"
 		fi
 	else
 		mysqldir=yes
@@ -336,7 +336,7 @@
 	else
 		AC_DEFINE(HAVE_MSQL, 1, [Have mSQL])
 		MSQL_CFLAGS="-I${msqldir}/include${msqldir_suffix}"
-		MSQL_LIBS="-L${msqldir}/lib${msqllibdir_suffix} -lmsql"
+		MSQL_LIBS="-L${libdir}${msqllibdir_suffix} -lmsql"
                 AC_MSG_CHECKING(for mSQL 3.x API)
                 AC_TRY_RUN([
                   #include "${msqldir}/include${msqldir_suffix}/msql.h"
@@ -402,7 +402,7 @@
 			AC_MSG_WARN(Postgres include files not found, backend not used)
 			postgresdir=""
 		fi
-		POSTGRES_LIBS="-L${postgresdir}/lib -lpq"
+		POSTGRES_LIBS="-L${libdir} -lpq"
 		AC_DEFINE(HAVE_POSTGRES, 1, [Have PostgreSQL])
 	fi
 fi
@@ -451,7 +451,7 @@
 		AC_MSG_WARN(tds backend not used)
 	else
 		AC_MSG_CHECKING(for freetds version)
-		freetdsvercheck="`grep \"^#define \(VERSION_NO \|TDS_VERSION_NO \)\" $freetds_incdir/tdsver.h|sed 's|^.*"\(.*\)"|\1|'`"
+		freetdsvercheck=$(grep "^#define \(VERSION_NO \|TDS_VERSION_NO \)" $freetds_incdir/tdsver.h|sed 's|^.*"\(.*\)"|\1|')
 		if test "x$freetdsvercheck" = "x"
 		then
 			AC_MSG_WARN(freetds version check failed - backend not used)
@@ -462,26 +462,26 @@
 				*v0.5*) AC_MSG_RESULT($freetdsvercheck - using v0.5x api)
 					AC_DEFINE(HAVE_FREETDS_VER0_5X, 1, [Have freetds v0.5x API])
 					FREETDS_CFLAGS="-I${freetds_incdir}"
-					FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+					FREETDS_LIBS="-L${libdir} -ltds"
 					AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 					;;
 				*v0.60) AC_MSG_RESULT($freetdsvercheck - using v0.60 api)
 					AC_DEFINE(HAVE_FREETDS_VER0_60, 1, [Have freetds v0.60 API])
 					FREETDS_CFLAGS="-I${freetds_incdir}"
-					FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+					FREETDS_LIBS="-L${libdir} -ltds"
 					AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 					;;
 				*v0.6[[3-9]]*) AC_MSG_RESULT($freetdsvercheck - using v0.63 api)
 					AC_DEFINE(HAVE_FREETDS_VER0_63, 1, [Have freetds v0.63 API])
 					AC_DEFINE(HAVE_FREETDS_VER0_6X, 1, [Have freetds v0.6x API])
 					FREETDS_CFLAGS="-I${freetds_incdir}"
-					FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+					FREETDS_LIBS="-L${libdir} -ltds"
 					AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 					;;
 				*v0.6*) AC_MSG_RESULT($freetdsvercheck - using v0.6x api)
 					AC_DEFINE(HAVE_FREETDS_VER0_6X, 1, [Have freetds v0.6x API])
 					FREETDS_CFLAGS="-I${freetds_incdir}"
-					FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+					FREETDS_LIBS="-L${libdir} -ltds"
 					AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 					;;
 				*)	AC_MSG_WARN($freetdsvercheck not supported - backend not used)
@@ -542,7 +542,7 @@
 		AC_DEFINE(HAVE_IBMDB2, 1, [Have IBM DB2])
 		AC_MSG_WARN(IBM DB2 Provider does not work yet)
 		IBMDB2_CFLAGS="-DIBMDB2_DIR=\\\"${ibmdb2dir}\\\" -I${ibmdb2dir}/include"
-		IBMDB2_LIBS="-L${ibmdb2dir}/lib -Wl,-rpath,${ibmdb2dir}/lib -ldb2"
+		IBMDB2_LIBS="-L${libdir} -Wl,-rpath,${libdir} -ldb2"
 	fi
 fi
 
@@ -601,8 +601,8 @@
 		AC_MSG_CHECKING(for Sybase host system type)
 		AC_MSG_RESULT($host)
 		case "$host" in
-		*linux*) sybase_ctlibs="-L${sybasedir}/lib -lct -lcs -lsybtcl -lcomn -lintl"
-			sybase_blklibs="-L${sybasedir}/lib -lblk ${sybase_ctlibs}"
+		*linux*) sybase_ctlibs="-L${libdir} -lct -lcs -lsybtcl -lcomn -lintl"
+			sybase_blklibs="-L${libdir} -lblk ${sybase_ctlibs}"
 			SYBASE_LIBS="${SYBASE_LIBS} ${sybase_blklibs}"
 			AC_CHECK_LIB(ct,tdsdump_on,[
 			AC_MSG_WARN(linking against freetds libct)
@@ -687,7 +687,7 @@
 	else
 		AC_DEFINE(HAVE_ORACLE, 1, [Have ORACLE])
 		ORACLE_CFLAGS=" -I$oracledir/include/oracle/client -I$oracledir/rdbms/demo -I${ORACLE_HOME}/rdbms/public -I${ORACLE_HOME}/plsql/public -I$oracledir/network/public"
-		ORACLE_LIBS="-L$oracledir/lib -lm -ldl -lclntsh"
+		ORACLE_LIBS="-L${libdir} -lm -ldl -lclntsh"
 	fi
 fi
 
@@ -725,7 +725,7 @@
 	else
 		AC_DEFINE(HAVE_FIREBIRD, 1, [Have FireBird])
 		FIREBIRD_CFLAGS="-I${firebirddir}/include"
-		FIREBIRD_LIBS="-L${firebirddir}/lib -lgds -ldl -lcrypt"
+		FIREBIRD_LIBS="-L${libdir} -lgds -ldl -lcrypt"
 	fi
 fi
 
@@ -763,7 +763,7 @@
 		AC_MSG_WARN(Xbase backend not used)
 	else
 		XBASE_CFLAGS=-I${xbasedir}/include
-		XBASE_LIBS="-L${xbasedir}/lib -lxbase"
+		XBASE_LIBS="-L${libdir} -lxbase"
 		AC_DEFINE(HAVE_XBASE, 1, [Have Xbase])
 	fi
 fi
@@ -793,7 +793,7 @@
 	mdbdir=""
 	for d in $dir /usr /usr/local /opt/gnome
 	do
-		if test -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql$libext -o -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql.a
+		if test -f $d/include/mdb/mdbtools.h -a -f $d/$lib/libmdbsql$libext -o -f $d/include/mdb/mdbtools.h -a -f $d/$lib/libmdbsql.a
 		then
 			AC_MSG_RESULT(found MDB Tools in $d)
 			mdbdir=$d
@@ -804,8 +804,8 @@
 	then
 		AC_MSG_WARN(MDB backend not used)
 	else
-		MDB_CFLAGS=-I${mdbdir}/include
-		MDB_LIBS="-L${mdbdir}/lib -lmdb -lmdbsql"
+		MDB_CFLAGS=-I${mdbdir}/include/mdb
+		MDB_LIBS="-L${libdir} -lmdb -lmdbsql"
 		AC_DEFINE(HAVE_MDB, 1, [Have MDB])
 	fi
 fi
@@ -844,7 +844,7 @@
 		AC_MSG_WARN(LDAP backend not used)
 	else
 		LDAP_CFLAGS="-I${ldapdir}/include"
-		LDAP_LIBS="-L${ldapdir}/lib -lldap -llber"
+		LDAP_LIBS="-L${libdir} -lldap -llber"
 		AC_DEFINE(HAVE_LDAP, 1, [Have LDAP])
 	fi
 fi
Tylko w libgda-1.9.103: configure.in.orig
Tylko w libgda-1.9.103: configure.in.rej
