diff -urN libgda-1.2.3.orig/providers/freetds/gda-freetds-provider.c libgda-1.2.3/providers/freetds/gda-freetds-provider.c
--- libgda-1.2.3.orig/providers/freetds/gda-freetds-provider.c	2006-02-12 16:09:22.000000000 -0200
+++ libgda-1.2.3/providers/freetds/gda-freetds-provider.c	2006-02-12 18:31:16.000000000 -0200
@@ -357,7 +357,9 @@
 		tds_cnc->database = NULL;
 	}
 	if (tds_cnc->config) {
-#ifdef HAVE_FREETDS_VER0_6X
+#ifdef HAVE_FREETDS_VER0_63
+		tds_free_connection (tds_cnc->config);
+#elif defined(HAVE_FREETDS_VER0_6X)
 		tds_free_connect (tds_cnc->config);
 #else
 		tds_free_config(tds_cnc->config);
@@ -802,7 +804,9 @@
 	}
 
 	/* there should not be any result tokens */
-#ifdef HAVE_FREETDS_VER0_6X
+#if defined(HAVE_FREETDS_VER0_63)
+	while ((tds_cnc->rc = tds_process_result_tokens (tds_cnc->tds, &tds_cnc->result_type, NULL))
+#elif defined(HAVE_FREETDS_VER0_6X)
 	while ((tds_cnc->rc = tds_process_result_tokens (tds_cnc->tds, &tds_cnc->result_type))
 #else
 	while ((tds_cnc->rc = tds_process_result_tokens (tds_cnc->tds)) 
diff -urN libgda-1.2.3.orig/providers/freetds/gda-freetds-recordset.c libgda-1.2.3/providers/freetds/gda-freetds-recordset.c
--- libgda-1.2.3.orig/providers/freetds/gda-freetds-recordset.c	2006-02-12 16:09:22.000000000 -0200
+++ libgda-1.2.3/providers/freetds/gda-freetds-recordset.c	2006-02-12 19:23:13.000000000 -0200
@@ -339,7 +339,15 @@
 	recset->priv->tds_cnc = tds_cnc;
 	recset->priv->res = tds_cnc->tds->res_info;
 
-#ifdef HAVE_FREETDS_VER0_6X
+#ifdef HAVE_FREETDS_VER0_63
+       while ((tds_cnc->rc = tds_process_result_tokens (tds_cnc->tds,
+	                                          &tds_cnc->result_type, NULL))
+              == TDS_SUCCEED) {
+               if (tds_cnc->result_type == TDS_ROW_RESULT) {
+                       gint row_type, compute_id;
+
+                       while ((tds_cnc->rc = tds_process_row_tokens(tds_cnc->tds, &row_type, &compute_id))
+#elif defined(HAVE_FREETDS_VER0_6X)
 	while ((tds_cnc->rc = tds_process_result_tokens (tds_cnc->tds,
 							 &tds_cnc->result_type))
 	       == TDS_SUCCEED) {
