diff -aurN libgda-1.9.100.orig/configure.in libgda-1.9.100/configure.in
--- libgda-1.9.100.orig/configure.in	2005-10-09 18:21:20.000000000 +0200
+++ libgda-1.9.100/configure.in	2005-10-19 21:05:15.000000000 +0200
@@ -778,7 +778,7 @@
 	mdbdir=""
 	for d in $dir /usr /usr/local /opt/gnome
 	do
-		if test -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql.so -o -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql
+		if test -f $d/include/mdb/mdbtools.h -a -f $d/$lib/libmdbsql.so -o -f $d/include/mdb/mmdbtools.h -a -f $d/$lib/libmdbsql
 		then
 			AC_MSG_RESULT(found MDB Tools in $d)
 			mdbdir=$d
@@ -789,7 +789,7 @@
 	then
 		AC_MSG_WARN(MDB backend not used)
 	else
-		MDB_CFLAGS=-I${mdbdir}/include
+		MDB_CFLAGS=-I${mdbdir}/include/mdb
 		MDB_LIBS="-L${mdbdir}/lib -lmdb -lmdbsql"
 		AC_DEFINE(HAVE_MDB, 1, [Have MDB])
 	fi
diff -aurN libgda-1.9.100.orig/providers/mdb/gda-mdb-provider.c libgda-1.9.100/providers/mdb/gda-mdb-provider.c
--- libgda-1.9.100.orig/providers/mdb/gda-mdb-provider.c	2005-09-22 12:24:01.000000000 +0200
+++ libgda-1.9.100/providers/mdb/gda-mdb-provider.c	2005-10-19 21:10:52.000000000 +0200
@@ -210,7 +210,7 @@
 	mdb_cnc = g_new0 (GdaMdbConnection, 1);
 	mdb_cnc->cnc = cnc;
 	mdb_cnc->server_version = NULL;
-	mdb_cnc->mdb = mdb_open (filename, MDB_WRITABLE);
+	mdb_cnc->mdb = mdb_open (filename);
 	if (!mdb_cnc->mdb) {
 		gda_connection_add_event_string (cnc, _("Could not open file %s"), filename);
 		g_free (mdb_cnc);
