--- libgda-3.0.1/configure.in.orig	2007-04-24 23:13:22.000000000 +0200
+++ libgda-3.0.1/configure.in	2007-05-13 22:19:38.395428965 +0200
@@ -197,7 +197,7 @@
 		else
 			AC_DEFINE(HAVE_BDB, 1, [Have Berkeley DB])
 			BDB_CFLAGS="-I${bdbdir}/include"
-			BDB_LIBS="-L${bdbdir}/lib ${bdblib}"
+			BDB_LIBS="-L${libdir} ${bdblib}"
 		fi
 	fi
 fi
@@ -252,7 +252,7 @@
 	else
 		AC_DEFINE(HAVE_ODBC, 1, [Have ODBC])
 		ODBC_CFLAGS="-I${odbcdir}/include"
-		ODBC_LIBS="-L${odbcdir}/lib ${odbclib}"
+		ODBC_LIBS="-L${libdir} ${odbclib}"
 	fi
 fi
 
@@ -308,7 +308,7 @@
 		else
 			AC_DEFINE(HAVE_MYSQL, 1, [Have MySQL])
 			MYSQL_CFLAGS="-I${mysqldir}/include${mysqldir_suffix}"
-			MYSQL_LIBS="-L${mysqldir}/lib${mysqllibdir_suffix} -lmysqlclient"
+			MYSQL_LIBS="-L${libdir}${mysqllibdir_suffix} -lmysqlclient"
 		fi
 	else
 		mysqldir=yes
@@ -367,7 +367,7 @@
 	else
 		AC_DEFINE(HAVE_MSQL, 1, [Have mSQL])
 		MSQL_CFLAGS="-I${msqldir}/include${msqldir_suffix}"
-		MSQL_LIBS="-L${msqldir}/lib${msqllibdir_suffix} -lmsql"
+		MSQL_LIBS="-L${libdir}${msqllibdir_suffix} -lmsql"
                 AC_MSG_CHECKING(for mSQL 3.x API)
                 AC_TRY_RUN([
                   #include "${msqldir}/include${msqldir_suffix}/msql.h"
@@ -433,7 +433,7 @@
 			AC_MSG_WARN(Postgres include files not found, backend not used)
 			postgresdir=""
 		fi
-		POSTGRES_LIBS="-L${postgresdir}/lib -lpq"
+		POSTGRES_LIBS="-L${libdir} -lpq"
 		AC_DEFINE(HAVE_POSTGRES, 1, [Have PostgreSQL])
 	fi
 fi
@@ -482,7 +482,7 @@
 		AC_MSG_WARN(tds backend not used)
 	else
 		AC_MSG_CHECKING(for freetds version)
-		freetdsvercheck="`grep \"^#define \(VERSION_NO \|TDS_VERSION_NO \)\" $freetds_incdir/tdsver.h|sed 's|^.*".*v\(.*\)"|\1|'`"
+		freetdsvercheck=$(grep "^#define \(VERSION_NO \|TDS_VERSION_NO \)" $freetds_incdir/tdsver.h|sed 's|^.*".*v\(.*\)"|\1|')
 		if test "x$freetdsvercheck" = "x"
 		then
 			AC_MSG_WARN(freetds version check failed - backend not used)
@@ -491,7 +491,7 @@
 		else
 			AC_MSG_RESULT($freetdsvercheck)
 			FREETDS_CFLAGS="-I${freetds_incdir}"
-			FREETDS_LIBS="-L${freetdsdir}/lib -ltds"
+			FREETDS_LIBS="-L${libdir} -ltds"
 			AC_DEFINE(HAVE_FREETDS, 1, [Have freetds])
 
 			freetds_version_major="`echo $freetdsvercheck|sed 's:^\([[^.]]*\).*:\1:'`"
@@ -568,7 +568,7 @@
 		AC_MSG_RESULT(found IBM DB2 in $ibmdb2dir)
 		AC_DEFINE(HAVE_IBMDB2, 1, [Have IBM DB2])
 		IBMDB2_CFLAGS="-DIBMDB2_DIR=\\\"${ibmdb2dir}\\\" -I${ibmdb2dir}/include"
-		IBMDB2_LIBS="-L${ibmdb2dir}/${ibmdb2libdir} -Wl,-rpath,${ibmdb2dir}/${ibmdb2libdir} -ldb2"
+		IBMDB2_LIBS="-L${ibmdb2dir}/${ibmdb2libdir} -Wl,-rpath,${libdir} -ldb2"
 	fi
 fi
 
@@ -627,8 +627,8 @@
 		AC_MSG_CHECKING(for Sybase host system type)
 		AC_MSG_RESULT($host)
 		case "$host" in
-		*linux*) sybase_ctlibs="-L${sybasedir}/lib -lct -lcs -lsybtcl -lcomn -lintl"
-			sybase_blklibs="-L${sybasedir}/lib -lblk ${sybase_ctlibs}"
+		*linux*) sybase_ctlibs="-L${libdir} -lct -lcs -lsybtcl -lcomn -lintl"
+			sybase_blklibs="-L${libdir} -lblk ${sybase_ctlibs}"
 			SYBASE_LIBS="${SYBASE_LIBS} ${sybase_blklibs}"
 			AC_CHECK_LIB(ct,tdsdump_on,[
 			AC_MSG_WARN(linking against freetds libct)
@@ -713,7 +713,7 @@
 	else
 		AC_DEFINE(HAVE_ORACLE, 1, [Have ORACLE])
 		ORACLE_CFLAGS=" -I$oracledir/include/oracle/client -I$oracledir/rdbms/demo -I${ORACLE_HOME}/rdbms/public -I${ORACLE_HOME}/plsql/public -I$oracledir/network/public"
-		ORACLE_LIBS="-L$oracledir/lib -lm -ldl -lclntsh"
+		ORACLE_LIBS="-L${libdir} -lm -ldl -lclntsh"
 	fi
 fi
 
@@ -751,7 +751,7 @@
 	else
 		AC_DEFINE(HAVE_FIREBIRD, 1, [Have FireBird])
 		FIREBIRD_CFLAGS="-I${firebirddir}/include"
-		FIREBIRD_LIBS="-L${firebirddir}/lib -lgds -ldl -lcrypt"
+		FIREBIRD_LIBS="-L${libdir} -lgds -ldl -lcrypt"
 	fi
 fi
 
@@ -789,7 +789,7 @@
 		AC_MSG_WARN(Xbase backend not used)
 	else
 		XBASE_CFLAGS=-I${xbasedir}/include
-		XBASE_LIBS="-L${xbasedir}/lib -lxbase"
+		XBASE_LIBS="-L${libdir} -lxbase"
 		AC_DEFINE(HAVE_XBASE, 1, [Have Xbase])
 	fi
 fi
@@ -819,7 +819,7 @@
 	mdbdir=""
 	for d in $dir /usr /usr/local /opt/gnome
 	do
-		if test -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql$libext -o -f $d/include/mdbtools.h -a -f $d/$lib/libmdbsql.a
+		if test -f $d/include/mdb/mdbtools.h -a -f $d/$lib/libmdbsql$libext -o -f $d/include/mdb/mdbtools.h -a -f $d/$lib/libmdbsql.a
 		then
 			AC_MSG_RESULT(found MDB Tools in $d)
 			mdbdir=$d
@@ -830,8 +830,8 @@
 	then
 		AC_MSG_WARN(MDB backend not used)
 	else
-		MDB_CFLAGS=-I${mdbdir}/include
-		MDB_LIBS="-L${mdbdir}/lib -lmdb -lmdbsql"
+		MDB_CFLAGS=-I${mdbdir}/include/mdb
+		MDB_LIBS="-L${libdir} -lmdb -lmdbsql"
 		AC_DEFINE(HAVE_MDB, 1, [Have MDB])
 
 		save_CFLAGS="$CFLAGS"
@@ -910,7 +910,7 @@
 		AC_MSG_WARN(LDAP backend not used)
 	else
 		LDAP_CFLAGS="-I${ldapdir}/include"
-		LDAP_LIBS="-L${ldapdir}/lib -lldap -llber"
+		LDAP_LIBS="-L${libdir} -lldap -llber"
 		AC_DEFINE(HAVE_LDAP, 1, [Have LDAP])
 	fi
 fi
