--- libgda-1.2.4/providers/xbase/gda-xbase-provider.c.orig	2007-01-09 13:50:55.000000000 +0100
+++ libgda-1.2.4/providers/xbase/gda-xbase-provider.c	2007-05-31 23:29:39.627378605 +0200
@@ -197,7 +197,7 @@
 		pdata->using_directory = FALSE;
 		xdb = gda_xbase_database_open (cnc, p_filename);
 		if (xdb) {
-			g_hash_table_insert (pdata->databases, gda_xbase_database_get_name (xdb), xdb);
+			g_hash_table_insert (pdata->databases, p_filename, xdb);
 		} else {
 			gda_xbase_provider_close_connection (provider, cnc);
 			return FALSE;
@@ -218,7 +218,7 @@
 			gchar *s = g_build_path (p_directory, p_filename);
 			xdb = gda_xbase_database_open (cnc, p_filename);
 			if (xdb) {
-				g_hash_table_insert (pdata->databases, gda_xbase_database_get_name (xdb), xdb);
+				g_hash_table_insert (pdata->databases, p_filename, xdb);
 			} else {
 				gda_connection_add_error_string (cnc, _("Could not open file %s"), s);
 			}
