--- libgda-1.0.3/providers/xbase/gda-xbase-database.cpp.orig	Fri Jan 17 01:18:10 2003
+++ libgda-1.0.3/providers/xbase/gda-xbase-database.cpp	Wed Apr 14 14:49:03 2004
@@ -19,7 +19,7 @@
  * Software Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
-#include <xdb/xbase.h>
+#include <xbase/xbase.h>
 #include "gda-xbase-database.h"
 #include "gda-xbase-provider.h"
 
@@ -104,3 +104,9 @@
 
 	return TRUE;
 }
+
+extern "C" const gchar *
+gda_xbase_database_get_name (GdaXbaseDatabase *xdb)
+{
+	return xdb->filename;
+}
--- libgda-1.0.3/providers/xbase/gda-xbase-database.h.orig	Tue Nov 26 00:27:23 2002
+++ libgda-1.0.3/providers/xbase/gda-xbase-database.h	Wed Apr 14 14:36:43 2004
@@ -34,6 +34,8 @@
 
 gboolean          gda_xbase_database_delete_all_records (GdaXbaseDatabase *xdb);
 
+const gchar *gda_xbase_database_get_name (GdaXbaseDatabase *xdb);
+
 G_END_DECLS
 
 #endif
--- libgda-1.0.3/configure.in.orig	Wed Apr 14 13:37:17 2004
+++ libgda-1.0.3/configure.in	Wed Apr 14 14:50:48 2004
@@ -665,7 +665,7 @@
 	xbasedir=""
 	for d in $dir /usr /usr/local /opt/xbase
 	do
-		if test -f $d/include/xdb/xbase.h
+		if test -f $d/include/xbase/xbase.h
 		then
 			AC_MSG_RESULT(found Xbase in $d)
 			xbasedir=$d
@@ -677,7 +677,7 @@
 		AC_MSG_WARN(Xbase backend not used)
 	else
 		XBASE_CFLAGS=-I${xbasedir}/include
-		XBASE_LIBS="-L${xbasedir}/lib -lxdb"
+		XBASE_LIBS="-L${xbasedir}/lib -lxbase"
 		AC_DEFINE(HAVE_XBASE)
 	fi
 fi
