diff -urN libgda-1.0.4.orig/providers/freetds/gda-freetds-provider.c libgda-1.0.4/providers/freetds/gda-freetds-provider.c
--- libgda-1.0.4.orig/providers/freetds/gda-freetds-provider.c	2004-04-19 10:11:12.126048784 +0200
+++ libgda-1.0.4/providers/freetds/gda-freetds-provider.c	2004-04-19 10:17:06.340200048 +0200
@@ -247,7 +247,7 @@
 	if (t_port)
 		tds_set_port(tds_cnc->login, atoi(t_port));
 	
-	tds_set_charset(tds_cnc->login, "iso_1");
+	tds_set_client_charset(tds_cnc->login, "iso_1");
 	tds_set_language(tds_cnc->login, "us_english");
 	tds_set_packet(tds_cnc->login, 512);
 
